lib/src/hierarchy_view.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2014, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library dump_viz.hierarchy_view;
+
+import 'dart:html';
+
+import 'package:polymer/polymer.dart';
+
+import 'info_helper.dart';
+import 'tree_table.dart';
+import 'view_version.dart';
+
+@CustomTag('hierarchy-view')
+class HierarchyView extends PolymerElement {
+  HierarchyView.created() : super.created() {}
+
+  TreeTable _treeTable;
```

Comment:
The polymer way to do this would be add an id on the element (lets say "treeTable" for example) and then just add a getter here that looks like the following:

'''dart
TreeTable get => $['treeTable'];
'''